gnomad_qc/v4/annotations/compute_coverage.py

```diff
@@ -515,12 +533,22 @@ def main(args):
             ht = ht.checkpoint(res.release_ht.path, overwrite=overwrite)
             ht.export(res.coverage_tsv)
 
-        if args.export_all_sites_an_release_tsv:
-            logger.info("Exporting all sites AN tsv...")
-            res = coverage_resources.export_all_sites_an_release_tsv
+        if args.export_all_sites_an_release_files:
+            logger.info("Exporting all sites AN HT...")
+            res = coverage_resources.export_all_sites_an_release_files
             res.check_resource_existence()
-            ht = res.allele_number_ht.ht()
-            ht = ht.select(AN=ht.AN[0])
+
+            ht = res.an_qual_hist_ht.ht()
```

Comment:
One basic question, I was looking for input resources, it seems that you used AN_qual_hist HT to get a new AN HT with regions flags now, why did we output a AN HT before separately?

Reply:
I need the qual hists for the combined freq creation, but we only want to release the all sites AN, we don't need to release the qual hists